app/models/user.rb

```diff
@@ -313,6 +316,9 @@ def update_profile_completed!
     update_profile_completed.save!
   end
 
+  # TODO: remove once slugs are live on frontend
+  before_validation(if: :name_changed?) { self.slug = name }
```

Comment:
Are we allowing spaces inside of slugs and other chars? Will that be ok for how we want to use this?